server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -385,13 +384,15 @@ private DivTag renderClientDateOfBirth(AccountModel account) {
     if (newestApplicant.isEmpty()) {
       return div();
     }
+    // TODO (#5503): Remove checking at WKP when removing feature flag
+    Optional<LocalDate> currentDobAtWKP =
+        newestApplicant.get().getApplicantData().getDateOfBirthAtWellKnownPath();
+    Optional<LocalDate> currentDobAtPIA = newestApplicant.get().getDateOfBirth();
```

Comment:
suggestion (non-blocking): Use `newestApplicant.get().getDateOfBirth()` directly instead of naming it `currentDobAtPIA`. I think it's clear enough what's going on and then you don't have to refactor it later.